src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -178,6 +178,8 @@ internal sealed partial class TdsParser
         internal bool isTcpProtocol { get; set; }
         internal string FQDNforDNSCache { get; set; }
 
+        private static readonly Encoding s_utf8EncodingWithoutBom = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
```

Comment:
Isn't there a common TdsParser.cs where this static can be placed?

Reply:
Yes. Moved it to common TdsParser.cs